include/misc/util.h

```diff
@@ -86,12 +86,12 @@ constexpr size_t ARRAY_SIZE(T(&)[N]) { return N; }
 #define INLINE
 #endif
 
-#ifndef max
-#define max(a, b) (((a) > (b)) ? (a) : (b))
+#ifndef MAX
+#define MAX(a, b) (((a) > (b)) ? (a) : (b))
```

Comment:
This will trip up the flash driver in fw-nrfconnect-nrf

Reply:
Flash dirver is now updated